capa/features/common.py

```diff
@@ -428,6 +428,20 @@ def __init__(self, value: str, description=None):
         super().__init__(value, description=description)
         self.name = "os"
 
+    def evaluate(self, ctx, **kwargs):
+        capa.perf.counters["evaluate.feature"] += 1
+        capa.perf.counters["evaluate.feature." + self.name] += 1
+
+        for feature, locations in ctx.items():
+            if not isinstance(feature, (OS,)):
+                continue
+
+            assert isinstance(feature.value, str)
+            if OS_ANY in (self.value, feature.value) or self.value == feature.value:
```

Comment:
interesting, so we support both a rule specifying `OS_ANY` and also an extractor emitting the feature `OS_ANY`. cool!

Reply:
not sure if this is how it was originally intended to work but I think it makes the most sense to work both ways